project.clj

```diff
@@ -94,10 +95,10 @@
                          [prismatic/plumbing "0.4.2"]
                          [prismatic/schema "1.1.9"]
 
-                         [puppetlabs/http-client "1.0.0"]
+                         [puppetlabs/http-client "1.1.0-SNAPSHOT"]
```

Comment:
I assume this PR should wait until we get these SNAPSHOT dependencies released?

Reply:
maybe? these form a circular dependency. They all need this clj-parent change too.

Reply:
If the snapshotted libraries blow up when used with this commit of clj-parent then we should probably bump all of them at once in this PR.

If they simply won't take advantage of the bouncycastle specified, then I think we should merge this PR without reference to them.

Reply:
The version declarations below will only impact projects that declare a dependency on the library and don't specify the version.  It should be safe to remove the snapshot versions and just release with the additions below.

Reply:
puppetdb and puppetserver do not specify the dependency versions, but they do specify the clj-parent version. The snapshot versions need to stay so that users of clj-parent pick up the appropriate http-client and such.

This is the real trouble with the circular dependency we have here. Basically, this change, jvm-ssl-utils and clj-http-client all need to merge at / around the same time.

Reply:
My understanding was that a SNAPSHOT version could be overridden by anything else locally. And wip commits to those repos will be published, at least internally, as those snapshot versions. 

This makes sense for your local dev workflow as you test these changes together, but I don't think its ever meaningful to release clj-parent with a snapshot version in it since it won't actually pin to a reproducible build of a dependency.

Reply:
Okay. I took the snapshot versions out. We'll just have to make another commit to clj-parent when we release the other jars, otherwise we won't be able to test that this is working properly.